content/Watsonx/WatsonxGov/Level-4/201.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: '201: Environment Setup'
-timeToComplete: 450
+timeToComplete: 45
```

Comment:
I think this is on purpose cause 450 is ~7.5 hours which includes the 6 hours waiting to provision ? Not sure.

Reply:
Ohhh.... right sorry, we should consider have it show up in house vs minutes when its this long. I will revert this in a later PR